lib/openstax/aws/stack.rb

```diff
@@ -65,9 +65,14 @@ def template
       end
     end
 
-    def create(params: {}, wait: false)
+    def create(params: {}, wait: false, skip_if_exists: false)
       logger.info("**** DRY RUN ****") if dry_run
 
+      if skip_if_exists and exists?
+        logger.info("Skipping #{name} stack - exists...")
+        return
```

Comment:
perhaps it makes sense to return something here?  like false or an error object?  from the caller perspective, it seems confusing to know if the expectation of create succeeded or not.

Reply:
the flag is basically for idempotent scripting: safely call to be sure it exists, either previously, or was created now. Note that this only happens when a dev explicitly uses an extra switch, so presumable will see the log, like this:
```
$ ./create_env  --env_name rjr-test --env_type dev --image_id ami-03945891ff05cc360 --do_it --try_harder
[2020-08-27 13:31:03.938] INFO  | Skipping rjr-test-quasar-network stack - exists...
[2020-08-27 13:31:04.020] INFO  | Skipping rjr-test-quasar-msk stack - exists...
[2020-08-27 13:31:04.110] INFO  | Skipping rjr-test-quasar-api stack - exists...
```

Reply:
what about the return value?  here...the caller of create has no idea if create succeeded for failed and for any reason.  do you feel this isnt necessary?